### ISSUE TITLE
Index Herbariorum sync is creating institutions and collections in GRSciColl of already existing institutions

### ISSUE THREAD

**Issue by EstebanMH-SiB** [gbif/portal-feedback#5056]
We are doing a review of our institutions and collections in GRSciColl and discovered that ih-sync is creating entries for already visible institutions.

For example this institution [Universidad Católica de Oriente](https://scientific-collections.gbif.org/institution/68f414e7-ebea-48a3-ad6d-38ee11ee2ca3) was created by us 2 years ago, and ih-sync created the same institution and herbarium 11 months ago [Universidad Católica de Oriente](https://scientific-collections.gbif.org/institution/7e8dae18-e60e-4edc-bdfe-eb6c07d92af5). 

Perhaps a validation can be put in place so new entries from ih-sync that have the same name of an institution or collection, will be put on hold until a manual review, that way we can avoid duplicates.

Thanks in advance,

**Comment by ManonGros** [comment]
Thanks @EstebanMH-SiB, so your suggestion would be to create new entries as suggestions to review?

That way a reviewer would be able to spot when there is a duplicate created and link the existing entries to IH instead. @marcos-lg what do you think? Would that be possible?

**Comment by marcos-lg** [comment]
That would be possible yeah. Would it be for all the new entries or just the ones whose name matches with an existing one?

**Comment by EstebanMH-SiB** [comment]
Yes, create the entries as suggestions and I think this should be done only to the ones whose name matches an already existing one, that way we avoid duplicates. With the new ones, they can proceed directly
